.github/workflows/container-image.yaml

```diff
@@ -40,9 +44,16 @@ jobs:
           TAG=$BRANCH
         fi
         docker login -u ${{ secrets.DOCKERHUB_USER }} -p ${{ secrets.DOCKERHUB_TOKEN }}
-        docker tag aws-efs-csi-driver $REPO:$TAG
-        docker push $REPO:$TAG
+
+        docker buildx build \
+              -t $REPO:$TAG \
+              --platform=linux/amd64,linux/arm64 \
+              --progress plain \
+              --push .
```

Comment:
`docker buildx` build and push are done as a single step.

Reply:
> The docker buildx build subcommand has a number of flags which determine where the final image will be stored. By default, i.e. if none of the flags are specified, the resulting image will remain captive in docker’s internal build cache. This is unlike the regular docker build command which stores the resulting image in the local docker images list. 

> --push: This flag tells docker to push the resulting image to a docker registry. This currently is the best way to store multi-architecture images.

https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408